src/Api/SentryInit.ts

```diff
@@ -22,6 +22,8 @@ export const initSentry = (sentryConfig: SentryConfig): void => {
     integrations: [
       new Sentry.ReactNativeTracing({
         tracingOrigins: ['localhost', strippedBaseUrl],
+        // NOTE: this is a workaround for Detox tests being stuck
```

Comment:
Can you link the relevant issue so we can understand where we could potentially revert the change?